src/puppeteer.mjs

```diff
@@ -24,9 +25,37 @@ async function getWords() {
       return result;
     });
 		await browser.close()
+
 	} catch (error) {
 		console.error(error)
 	}
 }
 
-getWords();
+function assembleList(){
+  assembled = []
+  filtered = set();
+
+  for (var i=0; i < 30; i++) {
+    assembled.append(getWords());
```

Comment:
wait... why are we calling `getWords`? Actually isn't `getWords` not being used anymore it looks like that runs puppeteer... Shouldn't we be just getting words from the .txt file